### PR TITLE
AST-1620 - WP blocks styles changes on preview of Tablet and mobile on Firefox when Astra theme is active.

### PR DESCRIPTION
**Pull request by GauravKhupse**
### Description
On the Firefox browser, the UAG blocks styles, fonts and width are getting changed when previewing the blocks on Tablet and Mobile view on the backend on It seems like the problem is with the Astra theme only. When I am switching it to some other themes, it is working fine.

Chrome preview is not working

Video - https://share.bsf.io/eDulKxR6

Discussion link: https://brainstormforce.slack.com/archives/CDS7TQ75Z/p1644399792268069

### Types of changes

 Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?

Added in the Card.

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -3,6 +3,7 @@ v3.7.10
 - Fix: Search element not displaying in Mobile Offcanvas.
 - Fix: Elementor Pro - WooCommerce Checkout Form's shipping details not look appropriate in editor mode as it looks on the frontend.
 - Fix: Gutenberg editor - Layout settings is not visble for Group block.
+- Fix: Fixed block editor style issue with the responsive preview option on the post or page.
```
Comment (imnavanath) [inline changelog.txt:6]:
```suggestion
- Fix: Theme's responsive styles not loading with the responsive preview option in the block editor.
```

**Review by imnavanath — APPROVED** [review]

**Comment by imnavanath** [comment]
Alongwith this PR please check typo font issue which is not loading in preview iframes - https://share.bsf.io/E0uy54O9

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: changelog.txt (hunk 1 above)
Comment (GauravKhupse) [reply changelog.txt]: Resolved!

File: inc/assets/js/block-editor-script.js [hunk 2]
```diff
@@ -117,3 +117,42 @@ function astra_onload_function() {
...
+				let iframe = preview.getElementsByTagName('iframe')[0];
+				let iframeDocument = iframe.contentWindow.document || iframe.contentDocument;
+
+				let element = iframeDocument.getElementById(
+					styleTagId
+				);
+				let elementBlock = iframeDocument.getElementById(
+					styleTagBlockId
+				);
+				if ( (null === element || undefined === element)&& (null === elementBlock || undefined === elementBlock )) {
```
[the remark this answers is not in the document]
Comment (GauravKhupse) [reply inc/assets/js/block-editor-script.js]: Changes done!

**Review by RajkiranBagal — APPROVED** [review]

**Review by nileshc-bsf — APPROVED** [review]

**Pull request merged**